tests/test_sqlalchemy.py

```diff
@@ -0,0 +1,221 @@
+        )[0]
+        assert isinstance(typ, sql_types.String)
+        # eq_(typ.length, 52)  #  No length in Databend
+
+
+class BooleanTest(_BooleanTest):
+    __backend__ = True
+
+    def test_whereclause(self):
+        # testing "WHERE <column>" renders a compatible expression
+        boolean_table = self.tables.boolean_table
+
+        with config.db.begin() as conn:
+            conn.execute(
+                boolean_table.insert(),
+                [
+                    {"id": 1, "value": True, "unconstrained_value": True},
+                    {"id": 2, "value": False, "unconstrained_value": False},
+                ],
+            )
+
+            eq_(
+                conn.scalar(
+                    select(boolean_table.c.id).where(boolean_table.c.value)
+                ),
+                1,
+            )
+            eq_(
+                conn.scalar(
+                    select(boolean_table.c.id).where(
+                        boolean_table.c.unconstrained_value
+                    )
+                ),
+                1,
+            )
+            # Databend does not support `WHERE NOT true|false`
+            # eq_(
+            #     conn.scalar(
+            #         select(boolean_table.c.id).where(~boolean_table.c.value)
+            #     ),
+            #     2,
+            # )
+            # eq_(
+            #     conn.scalar(
+            #         select(boolean_table.c.id).where(
+            #             ~boolean_table.c.unconstrained_value
+            #         )
+            #     ),
+            #     2,
+            # )
+
+
+class CompoundSelectTest(_CompoundSelectTest):
+
+    @testing.skip("databend")
+    def test_limit_offset_aliased_selectable_in_unions(self):
+        pass
+
+    @testing.skip("databend")
+    def test_limit_offset_selectable_in_unions(self):
+        pass
+
+    @testing.skip("databend")
+    def test_limit_offset_in_unions_from_alias(self):
+        pass
+
+
+class DeprecatedCompoundSelectTest(_DeprecatedCompoundSelectTest):
+    @testing.skip("databend")
+    def test_limit_offset_aliased_selectable_in_unions(self):
+        pass
+
+    @testing.skip("databend")
+    def test_limit_offset_selectable_in_unions(self):
+        pass
+
+    @testing.skip("databend")
+    def test_limit_offset_in_unions_from_alias(self):
+        pass
+
+
+class HasIndexTest(_HasIndexTest):
+    __requires__ = ('index_reflection',)
+
+
+class InsertBehaviorTest(_InsertBehaviorTest):
+    @testing.skip("databend")  # required autoinc columns
+    def test_insert_from_select_autoinc(self, connection):
+        pass
+
+    @testing.skip("databend")  # required autoinc columns
+    def test_insert_from_select_autoinc_no_rows(self, connection):
+        pass
+
+    @testing.skip("databend")  # required autoinc columns
+    def test_no_results_for_non_returning_insert(self, connection):
+        pass
+
+
+class LikeFunctionsTest(_LikeFunctionsTest):
```

Comment:
Skipped tests.

Reply:
We're just skipping the tests that require the escape clause. All the other tests in the ancestor class are relevant